README.md

```diff
@@ -1 +1,23 @@
-# edge-search-srs
\ No newline at end of file
+# edge-search-srs
+
+Copyright (C) 2018-2019 The Open Library Foundation
+
+This software is distributed under the terms of the Apache License,
+Version 2.0. See the file "[LICENSE](LICENSE)" for more information.
+
+## Introduction
+
+Microservice to allow 3rd party discovery services to perform FOLIO patron
+actions via the discovery service's UI.
+
```

Comment:
The README "Introduction" section looks like a copy of mod-patron. Is that correct?